client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -226,7 +226,7 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 			hideBack={ ! isCopySiteFlow( flow ) }
 			backLabelText={ __( 'Back to sites' ) }
 			hideSkip={ true }
-			flowName={ isCopySiteFlow( flow ) ? ( flow as string ) : 'linkInBio' }
+			flowName={ isCopySiteFlow( flow ) ? ( flow as string ) : ( flow as string ) }
```

Comment:
Could this be simplified, now that both cases will be the same? Not sure if it would still need the parenthesis.

```suggestion
			flowName={ ( flow as string ) }
```

Reply:
Absolutely! Thank you for reviewing it :)